tests/test-cases/Group6-VIC-Machine/6-16-Config.robot

```diff
@@ -278,6 +278,9 @@ Configure VCH resources
     Should Contain  ${output}  --memory-reservation=10
     Should Contain  ${output}  --memory-shares=163840
 
+    ${output}=  Run  bin/vic-machine-linux configure --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --timeout %{TEST_TIMEOUT} --storage-quota 1000
+    Should Contain  ${output} Storage quota exceeds datastore free space
```

Comment:
You're missing a space between `${output}` and `Storage quota exceeds datastore free space`.

A lot of these files don't follow the recommended formatting best practice, which is to use 4 spaces instead of 2 — that makes missing spaces much easier to spot. (https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#space-separated-format)

Reply:
Good catch. 4 spaces are great, but shall I follow all the existing 2 spaces style?